src/core/model_config.proto

```diff
@@ -1136,6 +1136,21 @@ message ModelWarmup
 //@@
 message ModelConfig
 {
+  //@@
+  //@@  .. cpp:var:: message ModelOperations
+  //@@
+  //@@     The metadata of libraries providing custom operations for this model.
+  //@@
+  message ModelOperations
```

Comment:
Please follow the established conventions. We do not use nested messages in ModelConfig. Create message at top-level as is done for other members of ModelConfig